test_scripts/LowVoltage/001_Low_Voltage_and_Wake_Up.lua

```diff
@@ -0,0 +1,123 @@
+---------------------------------------------------------------------------------------------------
+-- In case:
+-- 1) SDL is started (there was no LOW_VOLTAGE signal sent)
+-- 2) There are following app’s in HMI levels:
+-- App1 is in FULL
+-- App2 is in LIMITED
+-- App3 is in BACKGROUND
+-- App4 is in NONE
+-- 3) All apps have some data that can be resumed
+-- 4) SDL get LOW_VOLTAGE signal
+-- 5) And then SDL get WAKE_UP signal
+-- 6) All apps are registered with the same hashID
```

Comment:
does  `All apps are registered with the same hashID` mean there is only one hashid and all the apps use  the same id? probably not.
I think each app shall have its own hashid, when apps get disconnected and reconnect, each app uses the same id of their own

Reply:
@yang1070 Correct, each application has it's own `hashId`